src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -48,6 +48,11 @@ std::vector<std::shared_ptr<BundleSpecification>> GcsPlacementGroup::GetBundles(
   return ret_bundles;
 }
 
+size_t GcsPlacementGroup::GetBundleSize() const {
+  const auto &bundles = placement_group_table_data_.bundles();
+  return bundles.size();
```

Comment:
```suggestion
  return placement_group_table_data_.bundles().size();
```

Reply:
This method will be removed. It is unnecessary for now.

Reply:
I will remove this method. We don't need it for now.